packages/driver/src/cypress/source_map_utils.ts

```diff
@@ -63,14 +77,17 @@ const getSourcePosition = (filePath, position) => {
   if (!sourceMapConsumer) return null
 
   const sourcePosition = sourceMapConsumer.originalPositionFor(position)
+
   const { source, line, column } = sourcePosition
 
   if (!source || line == null || column == null) return
 
   // if the file is outside of the projectRoot
   // originalPositionFor will not provide the correct relative path
   // https://github.com/cypress-io/cypress/issues/16255
+  // @ts-ignore
```

Comment:
I wonder if `@ts-expect-error` is more appropriate here since we are accessing a private field of something we don't own. I would always expect this to be an issue so I doubt it would get removed anytime soon
```suggestion
  // @ts-expect-error
```